serving/samples/helloworld-java/README.md

```diff
@@ -170,8 +170,14 @@ folder) you're ready to build and deploy the sample app.
      for your app.
    - Automatically scale your pods up and down (including to zero active pods).
 
-1. To find the IP address for your service, use. If your cluster is new, it may
+1. To find the IP address for your service, use. 
```

Comment:
Not yours, but it would be great to update the sentence here.

s/use. /use the following command: